offline/packages/trackbase/autogen.sh

```
@@ -1,9 +1,8 @@
-#!/bin/sh
-srcdir=`dirname $0`
-test -z "$srcdir" && srcdir=.
+#!/ bin / sh
```

Shellcheck (0.11.0)

[error] On most OS, shebangs can only specify a single parameter.

(SC2096)